src/CentreonRemote/Tests/Infrastructure/Export/ExportParserJsonTest.php

```diff
@@ -62,10 +72,10 @@ public function testParse()
     /**
      * @covers \CentreonRemote\Infrastructure\Export\ExportParserJson::parse
      */
-    public function testParse2()
+    public function testParse2(): void
     {
         // add file
-        $this->fs->get('/tmp')->add('test1.json', new File('{"key":"val"}'));
+        $this->fs->get('/tmp')->add('test1.json', new File('{"key":"val"}')); /** @phpstan-ignore-line */
```

Comment:
To be discussed

Reply:
we can ignore this error cause we'll don't waste time to refactor this test